ProjectForce/force-app/main/default/classes/KanbanProjectApexClass.cls

```diff
@@ -0,0 +1,21 @@
+public with sharing class KanbanProjectApexClass {
+    @AuraEnabled
+    public static List<Backlog__c> getBacklogs(String project) {
+        return [
+            SELECT 
+                Id, Name, Story__c, NumberOfCardLabels__c,
```

Comment:
Are backlogs currently limited to three labels?

Reply:
yes.